src/applications/mhv-landing-page/utilities/data/resolveLinks.js

```diff
@@ -11,14 +14,31 @@ export const toggleLink = (link, featureToggles = {}) => {
     ariaLabel,
   } = link;
   const showNewLink =
-    (!!toggle && (hasOwn(featureToggles, toggle) && featureToggles[toggle])) ||
-    (!oldText && !oldHref);
+    isToggled(toggle, featureToggles) || (!oldText && !oldHref);
+
   const href = showNewLink ? newHref : oldHref;
   const text = showNewLink ? newText : oldText;
-  return { href, text, key: toggle, ariaLabel };
+  const isExternal = !!link.isExternal;
```

Comment:
I added this because otherwise, resolving an external link in `resolveLandingPageLinks` would result in the external link having the same styling as the "internal" links.

Reply:
And as far as I could tell, the **Payments** section is the only section that has an external link